app/webpacker/components/Results/TableCells.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Table } from 'semantic-ui-react';
+import CountryFlag from '../wca/CountryFlag';
+import EventIcon from '../wca/EventIcon';
+import { personUrl } from '../../lib/requests/routes.js.erb';
+import { formatAttemptResult } from '../../lib/wca-live/attempts';
+import { events } from '../../lib/wca-data.js.erb';
+
+export function CountryCell({ country }) {
+  return (
+    <Table.Cell textAlign="left">
+      {country.iso2 && <CountryFlag iso2={country.iso2} />}
+      {' '}
+      {country.name}
```

Comment:
See my remark about (not) using translated names in the Rankings PR

Reply:
see my remarks about this being a frontend country and not a backend one